src/gui/advancedsettings.cpp

```diff
@@ -197,7 +223,7 @@ void AdvancedSettings::updateCacheSpinSuffix(int value)
     if (value <= 0)
         spin_cache.setSuffix(tr(" (auto)"));
     else
-        spin_cache.setSuffix(tr(" MiB"));
+        spin_cache.setSuffix(" MiB");
 }
```

Comment:
@glassez:  This was requested by @sledgehammer999 in [<https://github.com/qbittorrent/qBittorrent/pull/3235#pullrequestreview-55943260>] (bottom comment):  

>`KiB`. 
>Also shouldn't it be non-translatable?